src/Operators/momentum_advection.jl

```diff
@@ -0,0 +1,64 @@
+####
+#### Momentum fluxes
+####
+
+@inline momentum_flux_uu(i, j, k, grid, u)    = ℑx_caa(i, j, k, grid, Ax_u, u) * ℑx_caa(i, j, k, grid, u)
```

Comment:
I wonder if it would be better to put this stuff in an `Advection` module. We can keep it here for now.

Reply:
Yeah seems premature when we only have one advection scheme.

Will be nice that a new advection scheme is just new definitions for the `momentum_flux` and `tracer_flux` functions. Hmmm, should probably rename `tracer_flux` to `advective_tracer_flux`.

Reply:
Definitely. There are other fluxes besides the advective flux for both momentum and tracers.